doc/manual/cookbooks/prescribed_velocity/prescribed_velocity.cc

```diff
@@ -0,0 +1,223 @@
+#include <deal.II/base/parameter_handler.h>
```

Comment:
is this the same file as in `/cookbooks`? If so, could you just reference the file `/cookbooks/prescribed_velocity/prescribed_velocity.cc` from `manual.tex`?